_posts/2018-03-01-Rust-1.24.1.md

```diff
@@ -0,0 +1,263 @@
+---
+layout: post
+title: "Announcing Rust 1.24.1"
+author: The Rust Core Team
+---
+
+The Rust team is happy to announce a new version of Rust, 1.24.1. Rust is a
+systems programming language focused on safety, speed, and concurrency.
+
+If you have a previous version of Rust installed via rustup, getting Rust
+1.24.1 is as easy as:
+
+```bash
+$ rustup update stable
+```
+
+If you don't have it already, you can [get `rustup`][install] from the
+appropriate page on our website, and check out the [detailed release notes for
+1.24.1][notes] on GitHub.
+
+[install]: https://www.rust-lang.org/install.html
+[notes]: https://github.com/rust-lang/rust/blob/stable/RELEASES.md#version-1241-2018-03-01
+
+## What's in 1.24.1 stable
+
+Several minor regressions were found in 1.24.0 which collectively merited a
+release.
+
+A quick summary of the changes:
+
+* Do not abort when unwinding through FFI (this reverts behavior added in 1.24.0)
+* Emit UTF-16 files for linker arguments on Windows
+* Make the error index generator work again
```

Comment:
If we can cut this that'd be great, it's really only relevant for distros AFAIK.

Reply:
it felt really weird to me with so few changes to leave one out